pkg/validation/app.go

```diff
@@ -174,6 +181,20 @@ func (v *Validator) validateName(ctx context.Context, cr v1alpha1.App) error {
 	return nil
 }
 
+// We make sure users cannot create in-cluster Apps outside their organization
+// or WC namespaces. Otherwise `.spec.namespace` could be exploited to override permissions.
+func (v *Validator) validateTargetNamespace(ctx context.Context, cr v1alpha1.App) error {
+	if key.InCluster(cr) {
+		if !strings.EqualFold(cr.Namespace, "giantswarm") {
```

Comment:
What is the benefit of using `strings.EqualFold` here?

Reply:
Hmm, I am afraid none, it was rather a random choice  😅

Reply:
I've changed it to inequality operator

Reply:
> I've changed it to inequality operator

Thanks ❤️ best to keep things simple.